README.rst

```diff
@@ -27,40 +27,31 @@ If you're currently using the ``azure`` package < 1.0 then please read important
 INSTALLATION
 ============
 
-You can install each library individually for each Azure service:
+You can install individually each library for each Azure service:
```

Comment:
I think the best phrasing might be "You can install each Azure service's library individually"